reference/common_errors.rst

```diff
@@ -24,16 +14,17 @@ Caused when the the coordinator node is unable to connect to a worker.
 
 ::
 
-  WARNING:  connection error: ec2-52-21-20-100.compute-1.amazonaws.com:5432
-  DETAIL:  no connection to the server
-  ERROR:  could not receive query results
+  ERROR:  connection to the remote node localhost:5432 failed with the following error: could not connect to server: Connection refused
+          Is the server running on host "localhost" (127.0.0.1) and accepting
+          TCP/IP connections on port 5432?
+
 
 Resolution
 ~~~~~~~~~~
 
 To fix, check that the worker is accepting connections, and that DNS is correctly resolving.
 
-Canceling the transaction since it was involved in a distributed deadlock
+Cancelling the transaction since it was involved in a distributed deadlock
```

Comment:
I think this is an American vs British spelling difference. You've chosen the British form, which might not match the rest of the document as well.

Reply:
Oh sorry about that, it's not that I prefer the British form, but I had never encountered "canceling" before. I will change it back to "canceling" in that case.